lib/foreman_maintain/cli/upgrade_command.rb

```diff
@@ -6,6 +6,11 @@ def self.target_version_option
                :required => false
       end
 
+      def self.disable_self_upgrade_option
+        option '--disable-self-upgrade', :flag, 'Disable foreman-maintain auto-upgrade',
```

Comment:
You are modifying package and command name but here it still shows `foreman-maintain`.